acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/runner/Eth2SignerRunner.java

```diff
@@ -88,17 +84,7 @@ public void start() {
   protected abstract void startExecutor(final List<String> params);
 
   public void shutdown() {
-    try {
-      shutdownExecutor();
-    } finally {
-      if (signerConfig.isHttpDynamicPortAllocation()) {
-        try {
-          MoreFiles.deleteRecursively(dataPath, RecursiveDeleteOption.ALLOW_INSECURE);
-        } catch (final IOException e) {
-          LOG.info("Failed to clean up temporary file: {}", dataPath, e);
-        }
-      }
-    }
+    shutdownExecutor();
```

Comment:
is it worth just renaming shutdownExecutor to "shutdown" and making it public?

Reply:
Prefer that the shutdownExecutor wasn't made public as the startup and shutdown are the public functions. For startup there is extra work to be done, but shutdown there isn't. So could get rid of the shutdownExecutor and rename that to shutdown but think it would be odd  to not have a shutdownExecutor. Maybe it doesn't matter though?